packages/fast-form-generator-react/src/form/form.utilities.ts

```diff
@@ -232,8 +278,9 @@ export function normalizeDataLocation(dataLocation: string, data: any): string {
 /**
  * Removes any references to array index
  */
-export function normalizeSchemaLocation(schemaLocation: string): string {
-    return schemaLocation.replace(squareBracketsRegex, "");
+export function normalizeSchemaLocation(schemaLocation: string, lastLocationItem?: boolean): string {
+    const schemaLocationWithoutArrayLocation: string = schemaLocation.replace(squareBracketsRegex, "");
+    return lastLocationItem ? schemaLocationWithoutArrayLocation.replace(arrayItemsRegex, "") : schemaLocationWithoutArrayLocation;
```

Comment:
I don't think you should add the `lastLocationItem` param here - it makes the function multi-purpose and kind of defeats the point of "normalizing" when you can get multiple outputs. It also makes harder to understand - reading the code, I now need  to go figure out what passing `true` does because it's not inherently descriptive.

I think you should preform the replacement contextually where you need it - if it crops up more than once I would *then* create a new abstraction instead of complicating this one.